rest-app/src/main/java/com/epam/repair/rest/controller/OrdersDTORestController.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.repair.rest.controller;
+
+import com.epam.repair.dao.OrdersDao;
+import com.epam.repair.model.dto.OrdersDTO;
+import com.epam.repair.service.mapper.OrdersMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class OrdersDTORestController {
+
+    @Autowired
+    private OrdersMapper ordersMapper;
```

Comment:
`OrderMapper`, it doesn't matter that it can convert lists, not only single order.

Reply:
OrderMapper, can convert order lists and a single order list.

Reply:
Comment accepted. OrdersMapper --> OrderMapper